v2/tests/log_sink_etw_int/log_sink_etw_int.c

```diff
@@ -0,0 +1,683 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include <stdio.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+#include "windows.h"
+#include "TraceLoggingProvider.h"
+#include "evntrace.h"
+#include "evntcons.h"
+
+#include "macro_utils/macro_utils.h"
+
+#include "c_logging/log_context_property_basic_types.h"
+#include "c_logging/log_context_property_type_ascii_char_ptr.h"
+#include "c_logging/log_context_property_value_pair.h"
+#include "c_logging/log_level.h"
+#include "c_logging/log_sink_if.h"
+#include "c_logging/log_context.h"
+#include "c_logging/logger.h"
+
+#include "c_logging/log_sink_etw.h"
+
+static size_t asserts_failed = 0;
+
+static GUID provider_guid = { 0xDAD29F36, 0x0A48, 0x4DEF, { 0x9D, 0x50, 0x8E, 0xF9, 0x03, 0x6B, 0x92, 0xB4 } };
+
+MU_DEFINE_ENUM_STRINGS(LOG_CONTEXT_PROPERTY_TYPE, LOG_CONTEXT_PROPERTY_TYPE_VALUES)
+
+#define POOR_MANS_ASSERT(cond) \
+    if (!(cond)) \
+    { \
+        (void)printf("%s:%d test failed\r\n", __FUNCTION__, __LINE__); \
+        asserts_failed++; \
+        abort(); \
```

Comment:
>abort [](http://example.com/codeflow?start=8&length=5)

So if this abort() ever executes, do we need asserts_failed too? Seeing as unless in a heavy threaded environment, at most assert_failed will get to is "1".

Reply:
We don't need the count of asserts any longer, I will remove it from this test suite and have a task to clean the other test suites in this repo.

Reply:
Actually changed it in all the test suites in this repo.